wordpress-mobile/WordPress-iOS
# [This Week Widget] Add default widget with supporting files and settings.

Ref #13044 

This is step 1 of adding the 'This Week' stat widget. That is, adding a default widget with supporting files and settings.

To test:

---
The main purpose of this is to ensure supporting files/settings are correct.
- `Signing & Capabilities`:
  - Uses the Bundle Identifier `org.wordpress.WordPressThisWeekWidget`.  
  - Uses the `WordPress This Week Widget` Provisioning Profiles.
- `*.entitlements`: same as for the other widgets.
- `Info*.plist`s - the only difference between the three is the `Bundle display name`. 
- `Localizable.strings` - is added for Base, and it empty for now.

Please review anything you can think of related to widget settings.

---
Add the widget to your Today view. Since I've simply added the default widget, you'll just see a `Hello World` message.

![default_widget](https://user-images.githubusercontent.com/1816888/70347194-37f2d200-181d-11ea-8688-bc9cf752f26e.jpeg)

---
PR submission checklist:
